pennylane/devices/legacy_facade.py

```diff
@@ -25,6 +25,7 @@
 import pennylane as qml
 from pennylane.measurements import MidMeasureMP, Shots
 from pennylane.transforms.core.transform_program import TransformProgram
+from pennylane.workflow.execution import INTERFACE_MAP
```

Comment:
isn't causing circular imports?

Reply:
Doesn't look like it. Everything passes.